index.html

```diff
@@ -32,7 +33,7 @@
     <section class="informative">
       <h2>Introduction</h2>
       <p>
-        Data portability is the ability of a user to move their data on one
+        Data portability is the ability of a user to move their data from one
```

Comment:
i think this edit was probably intended for the other PR (#9) and begs the same question Lisa had there about "system" versus "instance of any supporting server"